app/resources/api/v1/development_resource.rb

```diff
@@ -17,6 +17,8 @@ class DevelopmentResource < JSONAPI::Resource
       boolean_filters :rdv, :asofright, :ovr55, :clusteros, :phased,
         :stalled, :cancelled, :hidden
 
+      filter :status
```

Comment:
Name and municipality are still to come. This line does add project status.

I can add any filters you need to the Search Resource.